src/monodroid/jni/monodroid-glue.c

```diff
@@ -100,6 +101,21 @@ static int wait_for_gdb;
 static volatile int monodroid_gdb_wait = TRUE;
 static int android_api_level = 0;
 
+// Values correspond to the CPU_KIND_* macros
+static char* android_abi_names[] = {
+	"unknown",
+	"armeabi-v7a",
+	"arm64-v8a",
+	"mips",
+	"x86",
+	"x86_64",
+};
+#define ANDROID_ABI_NAMES_SIZE (sizeof(android_abi_names) / sizeof (char*))
```

Comment:
"conventional" is to use `(sizeof(android_abi_names) / sizeof (android_abi_names[0]))`, so that if the element type changes, `ANDROID_ABI_NAMES_SIZE` doesn't need to change.